src/frontend/src/tests/btc/utils/btc-transactions.utils.spec.ts

```diff
@@ -31,6 +31,7 @@ describe('mapBtcTransaction', () => {
 		const expectedResult = {
 			...mockBtcTransactionUi,
 			blockNumber: undefined,
+			confirmations: undefined,
```

Comment:
can it be `null`?  asking because the comment say

> // no confirmations - transaction status "pending"

Reply:
It can only be either `undefined` or a number. I rephrased the comment a bit.